src/FastExpressionCompiler/TestTools.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿#nullable disable
```

Comment:
Added `﻿#nullable disable` for every file, since this will be necessary for internal files, and you really don't use nullable types.